src/main/java/seedu/address/logic/security/Encrypt.java

```diff
@@ -0,0 +1,15 @@
+package seedu.address.logic.security;
+
+/**
+ * Encrypts data before saving
+ */
+public class Encrypt {
+
+    public static String encryptString(String plainString){
+        return plainString.concat("encrypt123456");
```

Comment:
You can consider using a Java Cryptography module

Reply:
Can refer to this: https://stackoverflow.com/questions/1132567/encrypt-password-in-configuration-files